target/linux/ipq40xx/Makefile

```diff
@@ -19,6 +19,6 @@ DEFAULT_PACKAGES += \
 	kmod-leds-gpio kmod-gpio-button-hotplug swconfig \
 	kmod-ath10k-ct wpad-basic-wolfssl \
 	kmod-usb3 kmod-usb-dwc3 ath10k-firmware-qca4019-ct \
-	ath10k-board-qca4019 uboot-envtools
+	uboot-envtools
```

Comment:
This will also remove it for the `chromium` subtarget. I think it is needed there.

Reply:
I think you can also add `-ath10k-board-qca4019` to the mikrotik subtarget.

Reply:
Indeed, this PR predated the Chromium target, let me check whether removing the package in mikrotik subtarget works as I think I did not work when this was initially developed.

Also, need to update the ath10k-ct as Ben merged the backport.

Reply:
Done, removing it in the subtarget appears to work(Probably forgot to purge tmp before).
ath10k-ct is updated instead of adding a patch.

It should be ready now.

Reply:
> I think you can also add `-ath10k-board-qca4019` to the mikrotik subtarget.

I am not sure if this will work with BUILDBOT config?
In a quick grep, I could not find any examples of packages being `-cut` from `DEFAULT_PACKAGES`, and I have run into issues with `DEVICE_PACKAGES` where `-cut` packages are ignored by BUILTBOT config for the initramfs images.

Reply:
Initramfs images with `CONFIG_TARGET_MULTI_PROFILE` and `CONFIG_TARGET_PER_DEVICE_ROOTFS` don't consider per device packages AFAIK, its a buildroot limitation.

I just tried adding `CONFIG_TARGET_MULTI_PROFILE` and `CONFIG_TARGET_PER_DEVICE_ROOTFS` and building for the MikroTik subtarget and it did not include the upstream board-2.bin in the squashfs

Reply:
Confirming that the buildbot snapshot initram image with these changes (r19677 5eee67a72f) does include, and use `ath10k-board-qca4019`
```
[    9.682562] ath10k_ahb a000000.wifi: board_file api 2 bmi_id 0:16 crc32 11892f9b
[   12.258448] ath10k_ahb a800000.wifi: board_file api 2 bmi_id 0:17 crc32 11892f9b
```
Would you, or @hauke be against a patch to remove `ath10k-board-qca4019` from the target makefile, and add it to the generic and chromium subtargets? I do testing from initram images (especially on unreachable devices), and would appreciate the more consistent wireless performance between the initram and installed images.

Thank you for your work on these changes!

Reply:
Thanks for checking, no idea why it worked as now with `CONFIG_BUILDBOT` it ignores the subtarget device package removal.
I don't have anything against that, it's probably the only way to do this properly.
If you chose to make a patch, can you send it to the mailing list, that is usually faster to get merged

Reply:
Otherwise I will make a patch since this is a mess that I created